src/vehicle.cpp

```diff
@@ -411,7 +412,15 @@ void vehicle::init_state(int init_veh_fuel, int init_veh_status)
         }
 
         if( pt.is_battery() ) {
-            pt.ammo_set( "battery", pt.ammo_capacity() * veh_fuel_mult / 100 );
+            if( veh_fuel_mult == 100 ) { // Mint condition vehicle
+                pt.ammo_set( "battery", pt.ammo_capacity() * veh_fuel_mult / 100 );
```

Comment:
If you know it's exactly 100, you don't need to multiply it by 100 to divide it by 100.